OSDOCS-8642: RN for CoreDNS filtration of  service queries for subdom… (#71457)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

==== Multi-network policy support for IPv6 networks
With this update, you can now create multi-network policies for IPv6 networks. For more information, see xref:../networking/multiple_networks/configuring-multi-network-policy.adoc#nw-multi-network-policy-ipv6-support_configuring-multi-network-policy[Supporting multi-network policies in IPv6 networks].

[id="ocp-4-15-CoreDNS-filtration-externalname-service-queries"]
==== CoreDNS filtration of ExternalName service queries for subdomains
As of {product-title} {product-version}, CoreDNS has been updated from 1.10.1 to 1.11.1.

> This update to CoreDNS resolved an issue where CoreDNS would incorrectly provide a response to a query for an `ExternalName` Service that shared its name with a top-level domain (e.g. com or org). A query for subdomains of an external service should not resolve to that external service. See the associated CoreDNS Github issue link:https://github.com/coredns/coredns/pull/6162[here] for more information.

Reviewer: ```suggestion
This update to CoreDNS resolved an issue where CoreDNS would incorrectly provide a response to a query for an `ExternalName` Service that shared its name with a top-level domain (e.g. com or org). A query for subdomains of an external service should not resolve to that external service. See the associated link:https://github.com/coredns/coredns/pull/6162[CoreDNS Github issue for more information.
```

Reviewer: See the IBM Style Guide for referring to web addresses. https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses#referring-to-web-addresses

Author: Updated As Suggested.

[id="ocp-4-15-ingress-dashboards"]
==== Ingress Operator metrics dashboard available
With this release, Ingress networking metrics are now viewable from within the {product-title} web console. See xref:../networking/networking-dashboards.adoc#ingress-dashboards[Ingress Operator dashboard] for more information.

> [id="ocp-4-15-CoreDNS-filtration-externalname-service-queries"]

Reviewer 2: I see that this renders fine, but I still think it's a best practice to leave a blank line in between the previous sentence and this ID line.

Author: Updated As Suggested.

==== CoreDNS filtration of ExternalName service queries for subdomains
As of {product-title} {product-version}, CoreDNS has been updated from 1.10.1 to 1.11.1.

> This update to CoreDNS resolved an issue where CoreDNS would incorrectly provide a response to a query for an `ExternalName` Service that shared its name with a top-level domain (e.g. com or org). A query for subdomains of an external service should not resolve to that external service. See the associated link:https://github.com/coredns/coredns/pull/6162[CoreDNS Github issue] for more information.

Reviewer 2: * I'm not sure if this still falls under needing to be okayed by a product experience rep (like Eric Rich) since it's referencing a PR or whatnot. But might need to check with @kalexand-rh to see if you do need to, if you haven't yet.
* A few minor grammatical suggestions

```suggestion
This update to CoreDNS resolved an issue where CoreDNS would incorrectly provide a response to a query for an `ExternalName` service that shared its name with a top-level domain, such as `com` or `org`. A query for subdomains of an external service should not resolve to that external service. See the associated link:https://github.com/coredns/coredns/pull/6162[CoreDNS GitHub issue] for more information.
```

Author: Updated As Suggested.
